### PR TITLE
Fix minor mistakes in the tests of dead-const-elim

### PR DESCRIPTION
**Pull request by Qining**

### CODE REVIEW

File: test/opt/test_eliminate_dead_const.cpp [hunk 1]
```diff
@@ -746,8 +746,8 @@ INSTANTIATE_TEST_CASE_P(
                 "%b_x = OpSpecConstantOp %int CompositeExtract %vec_b 0",
 
                 // Extract vector
-                "%a_xy = OpSpecConstantOp %v2int VectorShuffle %vec_a %vec_a 1 0",
```
Comment (antiagainst) [inline test/opt/test_eliminate_dead_const.cpp]: So essentially this is just because we name them as `a_xy` and `b_xy`?

**Comment by antiagainst** [comment]
+2

File: test/opt/test_eliminate_dead_const.cpp (hunk 1 above)
Comment (Qining) [inline test/opt/test_eliminate_dead_const.cpp]: Yes, just to make it consistent here.

**Comment by antiagainst** [comment]
LGTM. please squash before submtting.

**Pull request merged**